app/views/layouts/sumofus.slim

```diff
@@ -4,12 +4,14 @@ html
     title  = @page.title
     = render partial: 'shared/optimizely_snippet'
     meta name="viewport" content="width=device-width, initial-scale=1.0"
+    meta name="description" content="SumOfUs is a global movement of consumers, investors, and workers all around the world, standing together to hold corporations accountable for their actions and forge a new, sustainable and just path for our global economy. It's not going to be fast or easy. But if enough of us come together, we can make a real difference."
```

Comment:
I think yes, but let's leave that for a separate PR. It's not internationalized now on ActionSweet, so I'll create a separate ticket for it on Pivotal and we can bug @edubsky about getting a translation down the road.